scripts/background.js

```diff
@@ -1,3 +1,12 @@
+chrome.runtime.onInstalled.addListener(({ reason }) => {
+  if (reason === 'install') {
+    chrome.tabs.create({
+      url: 'https://decemberdevelopment.com/hide-amazon-cart',
+      active: true,
```

Comment:
`active: true` makes the new tab the active one instead of opening the tab in the background.